internal/rpc/user_devices.go

```diff
@@ -448,6 +459,17 @@ func (s *userDeviceRPCServer) deviceModelToAPI(ud *models.UserDevice) *pb.UserDe
 				Expiration: timestamppb.New(vc.ExpirationDate),
 			}
 		}
+
+		if sd := vnft.R.VehicleTokenSyntheticDevice; sd != nil {
```

Comment:
Could we do `sd != nil && !sd.TokenID.IsZero()`? Then we could get rid of the nesting below.